Breadcrumbs/Breadcrumbs.vue

```diff
@@ -14,43 +32,187 @@ const breadcrumbs = computed(() => {
     },
     ...breadcrumbsInjected.value,
   ]
-    .flatMap(breadcrumb => [breadcrumb, 'splitter'])
-    .slice(0, -1)
+})
+
+const breadcrumbEls = ref<(Element | ComponentPublicInstance | null)[]>([])
```

Comment:
It could be, according to TypeScript. It then complains here that types don't match

```
        <div
          v-if="isBreadcrumbVisibleByIndex![idx]"
          :ref="
            el => {
              breadcrumbEls[idx] = el
            }
          "
```